### DISCUSSION TITLE
Support for multiple poll questions in a single discussion

### DISCUSSION THREAD

**Discussion by brettcurtis** [community/community#122279] [Discussions]
### Select Topic Area

Product Feedback

### Body

Look like we can only have a single poll question per discussion. It would be great to have a single discussion with multiple questions. I want to use this for a very simple DevEx survey in my organization.

![image](https://github.com/community/community/assets/1610100/07fb1f32-cba9-4be1-a0e7-9338c0e7623d)

**Comment by oliversheridanmethven** [comment]
Agreed, on the face of it this looks quite easy to implement, and would be ideal for doing surveys.
